modules/imap/output_modules.php

```diff
@@ -782,8 +782,9 @@ protected function output() {
             $quota = $this->get('quota');
             $quota_max = $this->get('quota_max');
             if (!$folder && $quota) {
-                $res .= '<p class="quota_info"><div class="progress bg-secondary border"><div class="progress-bar bg-light" style="width:'.$quota.'%"></div></div>'.$quota.'% used on '.$quota_max.' MB</p>';
+                $res .= '<li class="quota_info"><div class="progress bg-secondary border"><div class="progress-bar bg-light" style="width:'.$quota.'%"></div></div>'.$quota.'% used on '.$quota_max.' MB</li>';
             }
+            $res .= '</ul>';
```

Comment:
It's working but i think i can put all in format_imap_folder_section like this:
```
$quota = $output_mod->get('quota');
$quota_max = $output_mod->get('quota_max');
if (!$folder && $quota) {
   $res .= '<p class="quota_info"><div class="progress bg-secondary border"><div class="progress-bar bg-light" style="width:'.$quota.'%"></div></div>'.$quota.'% used on '.$quota_max.' MB</p>';
   $res .= '<li class="quota_info"><div class="progress bg-secondary border"><div class="progress-bar bg-light" style="width:'.$quota.'%"></div></div>'.$quota.'% used on '.$quota_max.' MB</li>';
}
```
 and keep here only:
 ```
$res .= format_imap_folder_section($folder_data, $this->get('imap_expanded_folder_id'), $this, $with_input);
$this->out('imap_expanded_folder_formatted', $res);
```
Like that you don't need to move `</ul>`